LICENSE

```diff
@@ -1,7 +1,13 @@
 This project is licensed under the MIT license.
 
-Copyright (c) 2011 - 2014 Eloy Durán <eloy.de.enige@gmail.com>
-Copyright (c) 2012 - 2014 Fabio Pelosin <fabiopelosin@gmail.com>
+Copyright (c) 2011 - 2015 Eloy Durán <eloy.de.enige@gmail.com>, 
+                          Fabio Pelosin <fabiopelosin@gmail.com>,
+                          Samuel Giddins <segiddins@gmail.com>,
+                          Marius Rackwitz <git@mariusrackwitz.de">, 
+                          Kyle Fuller <inbox@kylefuller.co.uk>, 
```

Comment:
segiddins@segiddins.me, didn't realize there were emails

-Samuel E. Giddins

On Apr 16, 2015, at 9:10 AM, Orta <notifications@github.com> wrote:

In LICENSE:

> @@ -1,7 +1,13 @@
>  This project is licensed under the MIT license.
>  
> -Copyright (c) 2011 - 2014 Eloy Durán <eloy.de.enige@gmail.com>
> -Copyright (c) 2012 - 2014 Fabio Pelosin <fabiopelosin@gmail.com>
> +Copyright (c) 2011 - 2015 Eloy Durán <eloy.de.enige@gmail.com>, 
> +                          Fabio Pelosin <fabiopelosin@gmail.com>,
> +                          Samuel Giddins <segiddins@gmail.com>,
> +                          Marius Rackwitz <git@mariusrackwitz.de">, 
> +                          Kyle Fuller <inbox@kylefuller.co.uk>, 
ace

—
Reply to this email directly or view it on GitHub.